x-pack/solutions/observability/plugins/observability/dev_docs/navigation_test_plan.md

```diff
@@ -0,0 +1,58 @@
+# Navigation
+
+## Testing
+
+Navigation exists in three states for observability:
+
+1. Classic - Observability is one of several available solutions
+2. Cloud - Observability is the solution of focus
+3. Serverless - Observability plays a role, but is not the focus
+
+### Creating necessary environments
+
+Adding the following labels to your PR will trigger a deploy job in Buildkite:
+
+```
+ci:cloud-deploy
+ci:cloud-persist-deployment
+ci:project-deploy-elasticsearch
+```
+
+The output of the buildkite jobs will display instructions on how to connect to these environments.
+
+### Test cases
+
+#### Core navigations
+
+Each navigation solution has two core parts: side navigation, and breadcrumbs.
+The outcome of testing each part assumes the following:
+
+Side navigation opens and closes correctly, based on the deployment type (classic or cloud / serverless) and solution view.
+
+- Expect classic view to have all plugins displayed in a single column.
+- Expect cloud view to have a collapsed view of nav sections with headers, with each view visible in a flyout once the header is clicked on.
+- In serverless, observability plugins will become available in central navigation.
```

Comment:
We should probably figure out how to expose what SHOULD be represented in these navigations, easily. Maybe we can link to the config files for "solution view" and "serverless", at least? For classic, it might be trickier and we just have to mention that they need to make sure the nav matches with / without the variable "under test" for each test session?